### PR TITLE
consolidate redundant office hours documentation

### PR DESCRIPTION
**Pull request by ophie200**

### CODE REVIEW

**Review by fritz-astronomer — APPROVED** [review]

File: astro/office-hours.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ During an office hours meeting, you can:
 
 Before you book an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 
 
-To book office hours, click **Help** > **Book Office Hours** in the [Cloud UI](https://cloud.astronomer.io). Alternatively, you can click **Book office hours** in the left sidebar of the Astronomer documentation site.
+To book office hours, click **Book office hours** in the left sidebar of the Astronomer documentation site. This will take you to a Calendly form to schedule a 30-minute virtual meeting. In the form, provide details about any issues and questions that you want to discuss during the meeting.
```
Comment (lzdanski) [inline astro/office-hours.md:31]:
```suggestion
To book office hours, click **Book office hours** in the documentation sidebar navigation. Then, schedule a 30-minute virtual meeting on the Calendly form. In the form, provide details about any issues and questions that you want to discuss during the meeting.
```
- This page is in the documentation site, so we can assume users don't require that detail. 
- Spatial organization (left right top bottom above below) isn't a helpful cue for screen reader users, so we need a different term.

**Review by lzdanski — CHANGES_REQUESTED** [review]
@ophie200 - This looks good overall! I have some content edits for one part. LMK what you think and we can hopefully get these changes in today!

**Review by iamakashladdha — APPROVED** [review]
LGTM

File: astro/astro-support.md [hunk 2]
```diff
@@ -106,11 +106,8 @@ To add a teammate to an existing support request, cc them when replying on the s
 
 :::
 
-## Book office hours in the Cloud UI
+## Book office hours
 
-If you don't require break-fix support, Astronomer recommends scheduling a meeting with the Astronomer Data Engineering team during office hours. In an office hour meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines. Office hours are recommended if:
+If you don't require break-fix support, Astronomer recommends scheduling a meeting with the Astronomer Data Engineering team during office hours. In an office hour meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines.
```
Comment (lzdanski) [inline astro/astro-support.md:111]:
```suggestion
If you don't require break-fix support, Astronomer recommends scheduling a meeting with the Astronomer Data Engineering team during office hours. In an office hours meeting, you can ask questions, make feature requests, or get expert advice for your data pipelines. 
```

File: astro/office-hours.md [hunk 3]
```diff
@@ -28,6 +28,8 @@ During an office hours meeting, you can:
 
 Before you book an office hour appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question.
```
Comment (lzdanski) [inline astro/office-hours.md:29]:
```suggestion
Before you book an office hours appointment, please browse the documentation, [Astronomer Academy](https://academy.astronomer.io/), [Airflow Guides](https://docs.astronomer.io/learn/), or discussion on the [Astronomer Forum](https://forum.astronomer.io) to see if there are existing guides or resources that answer your question. 
```

**Review by lzdanski — APPROVED** [review]

**Pull request merged**